Updated commands to reflect Azure CLI 2.0 syntax

sjohner opened the pull request:
Updated commands for handling custom roles to reflect Azure CLI 2.0 syntax. Screenshots still show Azure CLI 1.0 commands and need to be updated as well.

azurecla commented:
Hi __@oranms__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (oranbr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Updated commands to reflect Azure CLI 2.0 syntax

Updated commands for handling custom roles to reflect Azure CLI 2.0 syntax. Screenshots still show Azure CLI 1.0 commands and need to be updated as well.
